pkg/apply/event/event.go

```diff
@@ -89,9 +89,20 @@ type ErrorEvent struct {
 	Err error
 }
 
+//go:generate stringer -type=WaitEventOperation -linecomment
+type WaitEventOperation int
+
+const (
+	ReconcilePending WaitEventOperation = iota // Pending
```

Comment:
A WaitEvent may be generated when a resource to be pruned has not reached the `NotFound` status. Is it okay using the `Reconcile` prefix for both apply and prune?

Reply:
I'm using "reconcile" to mean both here. Reconciled means the desired state is now the current state.